pkg/controllers/monitoring/monitoring-stack/conditions_test.go

```diff
@@ -13,6 +13,7 @@ func TestUpdateAvailable(t *testing.T) {
 	tt := []struct {
 		name               string
 		prometheus         monv1.Prometheus
+		alertmanager       *monv1.Alertmanager
```

Comment:
```suggestion
		alertmanager       monv1.Alertmanager
```

Reply:
Hi @sthaha! Any reason on why you resolved this conversation? :) I actually think it's a good point...it would make the tests more clear IMO

Reply:
I thought you might have wanted to set the variable to nil. I also wanted to do a proper review of the whole patch after checking it out locally.

Reply:
Yes....but I am thinking that it shouldn't be nil in every test...so I can update